client/components/navigation/Navbar.tsx

```diff
@@ -46,69 +53,172 @@ function Navbar() {
           </HStack>
         </Link>
 
-        <HStack spacing={{ base: 2, sm: 7 }}>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          borderRadius="18px"
-          href="/gigs">Gigs</Link>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          borderRadius="18px"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          href="/jobs">Jobs</Link>
+        <Button bg="gray.300" display={{ sm: 'block', md: 'none' }} zIndex={11}>
+          {isHamburgerOpen ? (
+            <CloseIcon
+              h={6}
+              w={6}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          ) : (
+            <HamburgerIcon
+              h={7}
+              w={7}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          )}
+        </Button>
+        {isHamburgerOpen && (
+          <VStack
+            position="fixed"
+            zIndex={10}
+            top={0}
+            left={0}
+            h="100%"
+            w="100%"
+            display={{ sm: 'flex', md: 'none' }}
+            bg="white"
+            spacing={10}
+            fontSize="3xl"
+            pt={20}
+          >
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
```

Comment:
We should prefer using [Chakra theme spacing values](https://chakra-ui.com/docs/theming/theme#spacing) over hard-coded values for two reasons:

- it creates consistent spacing across components
- the scale can be modified in the `theme` object later if we want

Reply:
We also shouldn't need to use `margin` or `padding` here anyway as the `spacing` prop for `VStack` allows us to control the amount of space between each element.

Reply:
Here I'm using the `padding` to give the hover background a button-like aspect. Without the padding, the background would just cover what's behind the letters and not the rest of each link. Does that make sense?

Link with padding:

![image](https://user-images.githubusercontent.com/53134194/142090220-6504e4e1-26a0-472f-a680-043a7f73f0c1.png)

Link without padding:

![image](https://user-images.githubusercontent.com/53134194/142090235-a13e398a-e7b2-4264-a90b-d01aef510fc7.png)